FppTest/struct/PrimitiveStructTest.cpp

```diff
@@ -39,8 +39,9 @@ class PrimitiveStructTest : public ::testing::Test {
     }
 
     void assertUnsuccessfulSerialization(Primitive& s, U32 bufSize) {
-        U8 data[bufSize];
-        Fw::SerialBuffer buf(data, sizeof(data));
+        // Avoid creating an array of size zero
+        U8 data[bufSize + 1];
+        Fw::SerialBuffer buf(data, bufSize);
```

Comment:
Was it intentional to not send `bufSize +1`?  Does `Fw::SerialBuffer` handle 0 length buffers in the constructor?

Reply:
I wanted to make the minimal change to avoid the compiler warning. Using `bufSize` as the size of the serial buffer (1) is the original behavior of the test code and (2) does not cause any warnings. A physical array size of zero causes a warning, so I added one to the array size.